src/_data/catalog/widgets.json

```diff
@@ -1298,6 +1518,17 @@
     "link": "https://api.flutter.dev/flutter/material/Radio-class.html",
     "image": "<img alt='' src='/assets/images/docs/widget-catalog/material-radio.png'>"
   },
+  {
+    "name": "Radio button",
+    "description": "Radio buttons allow users to select one option from a set.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
```suggestion
    "description": "Form component that app users can select or clear to select only one option from a set. Select the option to enable it or clear it to disable it.",
```